packages/tupaia-web/src/layout/MapLayout/MapLayout.tsx

```diff
@@ -3,41 +3,32 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import { MapWatermark } from './MapWatermark';
-
-const MapContainer = styled.div`
-  height: 100vh;
-  transition: width 0.5s ease;
-  width: 100%;
-`;
-
-export const Map = () => {
-  return <MapContainer>{/* <Map /> */}</MapContainer>;
-};
+import { Map } from './Map';
+import { TILE_SETS } from '../../constants';
+import { TilePicker } from '@tupaia/ui-map-components';
 
 const Wrapper = styled.div`
   flex: 1;
   display: flex;
   position: relative;
-  pointer-events: none;
-`;
-
-const MapControlsContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-  position: absolute; // make this absolutely positioned so that it lays over the map
-  width: 100%;
-  height: 100%;
 `;
 
+// Placeholder for legend
 const MapLegendWrapper = styled.div`
   display: flex;
   flex-direction: row;
   align-items: flex-end;
   justify-content: center;
+  position: absolute;
+  background-color: grey;
+  width: 300px;
+  height: 50px;
+  bottom: 1em;
+  left: calc(50% - 150px);
```

Comment:
```suggestion
  left: 50%;
  transform: translateX(-50%);
```
What do you think about this way of centering it?